agent/config.go

```diff
@@ -95,48 +90,52 @@ func (a Agent) identifyCurrentHost() error {
 		return agentError(err)
 	}
 
-
 	hostURL := index.Links.FindByRel("host-list")
 	hosts := []common.HostMessage{}
 	err = client.Get(hostURL, &hosts)
 	if err != nil {
 		return agentError(err)
 	}
+	log.Println("Retrieved hosts list, found", len(hosts), "hosts")
+
+	// find our host in the list of hosts
```

Comment:
The Romana CIDR should come from IPAM and/or topology. What are we trying to do here?